validation-test/stdlib/String.swift

```diff
@@ -999,6 +1000,30 @@ StringTests.test(
 #endif
 }
 
+StringTests.test("String._fastCStringContents()")
+  .skip(.nativeRuntime("_NSContiguousString undefined without _runtime(_ObjC)"))
+  .code {
+#if _runtime(_ObjC)
+  // ASCII tests
+  for i in (0..<128) {
+    let cs = _NSContiguousString(String(UnicodeScalar(i)!)._core)
```

Comment:
Actually, never mind, since the expectation below checks for non-nil, it has to be ASCII.